cmd/goci/main.go

```diff
@@ -163,7 +163,8 @@ func validateRun() error {
 	goModPath := "./go.mod"
 	fileBytes, err := os.ReadFile(goModPath)
 	if err != nil {
-		return fmt.Errorf("failed to read go.mod file: %v", err)
+		// If the go.mod file is not found, we will skip the go version check
```

Comment:
There is an error specifically for a file not existing that you can check against like this
```golang
if errors.Is(err, os.ErrNotExist) {
...
}
```